test/CMakeLists.txt

```diff
@@ -90,6 +105,12 @@ macro(onedpl_add_test test_source_file switch_off_checked_iterators)
     endif()
 
     target_include_directories(${_test_name} PRIVATE "${CMAKE_CURRENT_LIST_DIR}")
+    if (${is_std_test})
+        target_include_directories(${_test_name} PRIVATE "${CMAKE_CURRENT_LIST_DIR}/support_stdlib")
+    else()
+        target_include_directories(${_test_name} PRIVATE "${CMAKE_CURRENT_LIST_DIR}/support")
+    endif()
```

Comment:
Adding paths to  `${CMAKE_CURRENT_LIST_DIR}/support_stdlib` and `${CMAKE_CURRENT_LIST_DIR}/support` is not necessary because `${CMAKE_CURRENT_LIST_DIR}` is already there.
The current tests include test utilities as `"support/some_utility_header"`. You can remove this section and include the newly created headers as `"support_stdlib/some_utility_header"`. I have no strong objections against the current approach, though.

Reply:
This change has been rolled back.